src/Dapr.Client/DaprClientGrpc.cs

```diff
@@ -22,8 +22,8 @@ namespace Dapr.Client
     /// </summary>
     internal class DaprClientGrpc : DaprClient
     {
-        private readonly Autogenerated.Dapr.DaprClient client;
-        private readonly JsonSerializerOptions jsonSerializerOptions;
+        internal readonly Autogenerated.Dapr.DaprClient client;
```

Comment:
As I understand this is for validation in tests. expose it as internal get property.